### ISSUE TITLE
Upgrade to 1.995 on Ubuntu & Debian fails to install updated packaged

### ISSUE THREAD

**Issue by j7j7** [webmin/webmin#1680]
As reported previously the webmin update to 1.995 failed. I can issue the command to restart webmin, which gets it back up and running.
However, I since had some general package updates showing, if I click via the GUI to update these packages - it fails, because I think it tries to continue with the webmin update, which calls webmin to stop, then the package doesn't update.

<img width="906" alt="Screenshot 2022-07-04 at 8 50 17 AM" src="https://user-images.githubusercontent.com/31825378/177064010-1e7eb744-4755-4666-928a-04c7777e9d76.png">

Restarting webmin via 'systemctl stop webmin' and 'systemctl start webmin' - gets webmin back working, but still shows the other packages still needing updating.

I *think* webmin is still 'half' updated, is there any command line to force the update without it trying to shutdown?

**Comment by unknown10777** [comment]
I have configured webmin repo in apt so I use `apt install webmin`
![image](https://user-images.githubusercontent.com/83267603/177135155-2503fc57-0d7e-48ae-9a1e-cc05faccc5fd.png)

**Comment by iliajie** [comment]
> I think webmin is still 'half' updated, is there any command line to force the update without it trying to shutdown? 

Unfortunately, there was a bug in Webmin 1.995 that simply stopped Webmin process when upgrading via UI.

The work around for now is to install packages upgrade via console this time. 

> I have configured webmin repo in apt so I use `apt install webmin`

This mustn't be necessary on future upgrades, after Webmin 1.996 is released.

**Comment by etiennedupuis** [comment]
> As reported previously the webmin update to 1.995 failed. I can issue the command to restart webmin, which gets it back up and running. However, I since had some general package updates showing, if I click via the GUI to update these packages - it fails, because I think it tries to continue with the webmin update, which calls webmin to stop, then the package doesn't update.
> 
> <img alt="Screenshot 2022-07-04 at 8 50 17 AM" width="906" src="https://user-images.githubusercontent.com/31825378/177064010-1e7eb744-4755-4666-928a-04c7777e9d76.png">
> 
> Restarting webmin via 'systemctl stop webmin' and 'systemctl start webmin' - gets webmin back working, but still shows the other packages still needing updating.
> 
> I _think_ webmin is still 'half' updated, is there any command line to force the update without it trying to shutdown?

I have the same problem when updating php 8.1 latest update. Hopefully, webmin will update to version 1.996 soon. I still havent been prompted for the update (I'm using debian 10)

**Comment by etiennedupuis** [comment]
I had to run `sudo dpkg --configure -a` to get version 1.996 available.

**Comment by iliajie** [comment]
Yes, correct and sorry about that. Upcoming Webmin 1.997 will have all those issues sorted!
